Add AI features prerequisite to dbt MCP OAuth docs (#9075)
fishtown-analytics/docs.getdbt.com: website/snippets/_mcp-qs-platform.md

- [Install uv](https://docs.astral.sh/uv/getting-started/installation/)
- A [<Constant name="dbt_platform"/> account](https://www.getdbt.com/signup)

> - If you are using OAuth, your account admin has enabled AI features on your <Constant name="dbt_platform"/> account. Refer to [Enable dbt Copilot](/docs/platform/enable-dbt-copilot) for more info.

Author: ```suggestion
- If you're using OAuth, your account admin has to enable AI features on your <Constant name="dbt_platform"/> account. Refer to [Enable dbt Copilot](/docs/platform/enable-dbt-copilot) for more info.
```

<StaticSubdomainRequired />

:::note AI features required for OAuth

> OAuth authentication requires AI features to be enabled on your dbt Cloud account. An account admin must enable this before you can complete OAuth setup. See [Enable dbt Copilot](/docs/platform/enable-dbt-copilot) for instructions.

Author: ```suggestion
OAuth authentication requires AI features to be enabled on your <Constant name="dbt_platform"/> account. An account admin must enable this before you can complete OAuth setup. Refer to [Enable dbt Copilot](/docs/platform/enable-dbt-copilot) for more info.
```

> OAuth authentication requires AI features to be enabled on your <Constant name="dbt_platform"/> account. An account admin must enable this before you can complete OAuth setup. Refer to [Enable dbt Copilot](/docs/platform/enable-dbt-copilot) for more info.

Reviewer: ```suggestion
OAuth for MCP requires an account admin enable AI features on your <Constant name="dbt_platform"/> account before you can complete setup. Refer to [Enable dbt Copilot](/docs/platform/enable-dbt-copilot) for more info.
```
